pkg/resources/external_oauth_integration.go

```diff
@@ -41,548 +46,777 @@ var oauthExternalIntegrationSchema = map[string]*schema.Schema{
+					return diag.FromErr(err)
+				}
+			}
+			if v := d.GetRawConfig().AsValueMap()["external_oauth_audience_list"]; !v.IsNull() {
+				if err = d.Set("external_oauth_audience_list", ctyValToSliceString(v.AsValueSlice())); err != nil {
+					return diag.FromErr(err)
+				}
+			}
+			if v := d.GetRawConfig().AsValueMap()["external_oauth_any_role_mode"]; !v.IsNull() {
+				if err = d.Set("external_oauth_any_role_mode", v.AsString()); err != nil {
+					return diag.FromErr(err)
+				}
+			}
+			if v := d.GetRawConfig().AsValueMap()["external_oauth_scope_delimiter"]; !v.IsNull() {
+				if err = d.Set("external_oauth_scope_delimiter", v.AsString()); err != nil {
+					return diag.FromErr(err)
+				}
+			}
+			if v := d.GetRawConfig().AsValueMap()["external_oauth_scope_mapping_attribute"]; !v.IsNull() {
+				if err = d.Set("external_oauth_scope_mapping_attribute", v.AsString()); err != nil {
+					return diag.FromErr(err)
+				}
+			}
+			if v := d.GetRawConfig().AsValueMap()["comment"]; !v.IsNull() {
+				if err = d.Set("comment", v.AsString()); err != nil {
+					return diag.FromErr(err)
+				}
+			}
 		}
-	}
-	if d.HasChange("blocked_roles") {
-		val, ok := d.GetOk("blocked_roles")
-		if ok {
-			alterInput.ExternalOauthBlockedRolesList = expandStringList(val.(*schema.Set).List())
-			alterInput.ExternalOauthBlockedRolesListOk = true
-			runAlter = true
-		} else {
-			unsetInput.ExternalOauthBlockedRolesListOk = true
-			runUnset = true
+
+		if err = d.Set(ShowOutputAttributeName, []map[string]any{schemas.SecurityIntegrationToSchema(integration)}); err != nil {
+			return diag.FromErr(err)
 		}
-	}
-	if d.HasChange("allowed_roles") {
-		val, ok := d.GetOk("allowed_roles")
-		if ok {
-			alterInput.ExternalOauthAllowedRolesList = expandStringList(val.(*schema.Set).List())
-			alterInput.ExternalOauthAllowedRolesListOk = true
-			runAlter = true
-		} else {
-			unsetInput.ExternalOauthAllowedRolesListOk = true
-			runUnset = true
+
+		if err = d.Set(DescribeOutputAttributeName, []map[string]any{schemas.ExternalOauthSecurityIntegrationPropertiesToSchema(integrationProperties)}); err != nil {
+			return diag.FromErr(err)
 		}
-	}
-	if d.HasChange("audience_urls") {
-		val, ok := d.GetOk("audience_urls")
-		if ok {
-			alterInput.ExternalOauthAudienceList = expandStringList(val.(*schema.Set).List())
-			alterInput.ExternalOauthAudienceListOk = true
-			runAlter = true
-		} else {
-			unsetInput.ExternalOauthAudienceListOk = true
-			runUnset = true
+
+		params, err := client.Parameters.ShowParameters(ctx, &sdk.ShowParametersOptions{
+			Like: &sdk.Like{
+				Pattern: sdk.Pointer("%%EXTERNAL_OAUTH%%"),
+			},
+			In: &sdk.ParametersIn{
+				Account: sdk.Pointer(true),
+			},
+		})
+		if err != nil {
+			return diag.FromErr(err)
 		}
+		if err = d.Set(ParametersAttributeName, []map[string]any{schemas.ExternalOauthParametersToSchema(params)}); err != nil {
```

Comment:
let's change the parametrs to some other list, significant_parameters? connected_parameters? or something like that

Reply:
Done.